contracts/axelar-interchain-token-service/src/storage_types.rs

```diff
@@ -1,17 +1,9 @@
 use soroban_sdk::{contracttype, String};
 
-#[contracttype]
-#[derive(Clone, Debug)]
-pub struct TrustedItsAddress {
-    pub chain_name: String,
-    pub trusted_address: String,
-}
-
 #[contracttype]
 #[derive(Clone, Debug)]
 pub enum DataKey {
     Initialized,
-    AuthModule,
     Owner,
-    TrustedAddress(TrustedItsAddress),
-}
\ No newline at end of file
+    TrustedAddress(String),
```

Comment:
```suggestion
    TrustedAddress {
        chain: String
    },
```
can we use named args instead? It might be a bit less efficient if it's getting XDR encoded though, something to investigate as part of determining the storage pattern